### PR TITLE
Gwtii 263 tms profiles

### PR DESCRIPTION
**Pull request by jandm**

### CODE REVIEW

**Comment by jandm** [comment]
Example deployed here:
http://dev.geomajas.org/GWTII-263-Tms/?sample=tmsprofile

**Comment by oliverm-be** [comment]
+1 (binding)

File: plugin/tms/tms/src/main/java/org/geomajas/gwt2/plugin/tms/client/TmsClient.java [hunk 1]
```diff
@@ -59,6 +111,63 @@ public static TmsClient getInstance() {
...
+	 */
+	public MapConfiguration createTmsMap(Profile profile, int nrOfZoomLevels) {
+		switch (profile) {
+			case GLOBAL_GEODETIC:
+				return createTmsMap(profile, "EPSG:4326", CrsType.DEGREES, new Bbox(-180, -90, 360, 180), 256,
+						nrOfZoomLevels);
+			default:
+				return createTmsMap(profile, "EPSG:3857", CrsType.METRIC, new Bbox(-MERCATOR, -MERCATOR, 2 * MERCATOR,
+						2 * MERCATOR), 256, nrOfZoomLevels);
+		}
```
Comment (janvenstermans) [inline plugin/tms/tms/src/main/java/org/geomajas/gwt2/plugin/tms/client/TmsClient.java]: and if the profile is local, with a custom EPSG ? Shouldn't it be "not implemented yet"?

Comment (jandm) [inline plugin/tms/tms/src/main/java/org/geomajas/gwt2/plugin/tms/client/TmsClient.java]: For local profiles you should directly call the createTmsMap() method with all arguments. For local profiles you should directly call the createTmsMap() method with all arguments. I will add an exception and a specific method for local profiles.

**Pull request merged**